src/Foundation/NSDictionary_2.cs

```diff
@@ -147,7 +147,28 @@ public TValue[] ObjectsForKeys (TKey[] keys, TValue marker)
 			return Runtime.GetNSObject<NSDictionary<TKey,TValue>> (_FromObjectsAndKeysInternal (objects.Handle, keys.Handle));
 		}
 
-		public static NSDictionary<TKey,TValue> FromObjectsAndKeys (TKey [] objects, TValue [] keys)
+		public static NSDictionary<TKey, TValue> FromObjectsAndKeys (TValue [] objects, TKey [] keys, nint count)
```

Comment:
@rolfbjarne yes, code compiled with C8 might not work on C7 :( but that's not a level of compatibility we (have or can) guarantee.